### PR TITLE
Return dataframes, not BQL cursors, for describe.

### PR DESCRIPTION
**Pull request by fsaad**

### CODE REVIEW

File: src/bql_utils.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def query(bdb, bql):
 
 
 def describe_table(bdb, table_name):
-    """Returns a SQLite3 cursor containg description of `table_name`.
+    """Returns a DataFrame cursor containg description of `table_name`.
```
Comment (gregory-marton) [inline src/bql_utils.py]: Omit the word "cursor" here.

File: src/bql_utils.py [hunk 2]
```diff
@@ -184,11 +184,12 @@ def describe_table(bdb, table_name):
...
             ORDER BY tabname ASC, colno ASC
         '''
-    return bdb.sql_execute(sql, bindings=(table_name,))
+    curs = bdb.sql_execute(sql, bindings=(table_name,))
+    return cursor_to_df(curs)
 
 
 def describe_generator(bdb, generator_name):
-    """Returns a SQLite3 cursor containg description of `generator_name`.
+    """Returns a DataFrame containg description of `generator_name`.
```
Comment (gregory-marton) [inline src/bql_utils.py]: containg --> containing.

**Comment by gregory-marton** [comment]
LG with minor nits. Please merge after fixing typos.

**Comment by fsaad** [comment]
Addressed comments in e9ff201a59c75c1468be6efcc0f45ab70361cb52

**Pull request merged**